public/app/features/alerting/unified/components/AlertLabelDropdown.tsx

```diff
@@ -23,6 +54,8 @@ const AlertLabelDropdown: FC<AlertLabelDropdownProps> = React.forwardRef<HTMLDiv
             backspaceRemovesValue={false}
             onChange={onChange}
             onOpenMenu={onOpenMenu}
+            filterOption={customFilter}
```

Comment:
Isn't `createFilter({ ignoreCase: false });` enough for our case? Do we need `customFilter` function?

Reply:
no, because then we get this error:
```
Type '(option: FilterOptionOption<unknown>, rawInput: string) => boolean' is not assignable to type '(option: SelectableValue<string>, searchQuery: string) => boolean'.
  Types of parameters 'option' and 'option' are incompatible.
    Property 'data' is missing in type 'SelectableValue<string>' but required in type 
```